modules/compliance-supported-profiles.adoc

```diff
@@ -8,86 +8,106 @@
 The Compliance Operator provides the following compliance profiles:
 
 .Supported compliance profiles
-[cols="10%,40%,10%,40%", options="header"]
+[cols="10%,40%,10%,40%,10%", options="header"]
 
 |===
 |Profile
 |Profile title
-|Compliance Operator version 
+|Compliance Operator version
 |Industry compliance benchmark
+|Supported architectures
 
 |ocp4-cis
 |CIS Red Hat OpenShift Container Platform 4 Benchmark
 |0.1.39+
 |link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnote:cisbenchmark[To locate the CIS RedHat OpenShift Container Platform v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on *Kubernetes* and then *Download Latest CIS Benchmark*, where you can then register to download the benchmark.]
+|`x86_64`
```

Comment:
You could potentially remove the back-ticks from all of the supported architecture references in this column. My understanding is that they are short-hand architecture names and not direct code references, in this context.